routing/endpointregistry_test.go

```diff
@@ -50,20 +54,34 @@ func TestSetAndGetAnotherKey(t *testing.T) {
 }
 
 func TestDoRemovesOldEntries(t *testing.T) {
```

Comment:
Lets change this file package to `routing_test` since it tests only exported interfaces.
We can add SetNow to export_test.go